js/Book.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable linebreak-style */
+class Book {
+  static #DATA_KEY = 'books';
+
+  static #booksDataFromLocalStorage;
+
+  constructor(bookId, bookTitle, bookAuthor) {
+    this.bookId = bookId;
+    this.bookTitle = bookTitle;
+    this.bookAuthor = bookAuthor;
+  }
+
+  static #getBooksDataFromLocalStorage() {
+    return JSON.parse(localStorage.getItem(this.#DATA_KEY));
+  }
+
+  static #saveBooksDataToLocalStorage(data) {
+    localStorage.setItem(this.#DATA_KEY, JSON.stringify(data));
+  }
+
+  static get getStoredDataFromStorage() {
+    return this.#getBooksDataFromLocalStorage();
+  }
+
+  static addBookToStorage(bookDetails) {
+    this.#booksDataFromLocalStorage = this.#getBooksDataFromLocalStorage();
+
+    if (!this.#booksDataFromLocalStorage) {
+      this.newData = [];
+      this.newData.push(bookDetails);
+      this.#saveBooksDataToLocalStorage(this.newData);
+      return;
+    }
+
+    this.updatedData = [...this.#getBooksDataFromLocalStorage()];
+    this.updatedData.push(bookDetails);
+    this.#saveBooksDataToLocalStorage(this.updatedData);
+  }
+
+  static removeBookFromStorage(bookId) {
+    const dataFromStore = this.#getBooksDataFromLocalStorage();
+    const updatedData = dataFromStore.filter((book) => book.bookId !== bookId);
+    this.#saveBooksDataToLocalStorage(updatedData);
```

Comment:
- [ ] Good job so far by refactoring your code :clap: But I noticed that when there is a piece of duplicate information when I tried to remove one, all the duplicate pieces of information are removed. Please fix it in order to make it possible to remove the information we need to remove and make your application work as expected :+1: